packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -455,7 +458,7 @@ class AnimationController extends Animation<double>
   @override
   bool get isAnimating => _ticker != null && _ticker!.isActive;
 
-  _AnimationDirection _direction;
+  _AnimationDirection _direction = _AnimationDirection.forward;
```

Comment:
Both constructors initialize `_direction` to the same value, so this can be pulled out of the constructor logic.